src/Components/Telemetry.md

```diff
@@ -38,6 +38,16 @@ Aspire.Azure.Messaging.EventHubs:
 - Metric names:
   - none (currently not supported by the Azure SDK)
 
+Aspire.Azure.Messaging.WebPubSub:
+- Log categories:
+  - "Azure.Core"
+  - "Azure.Identity"
+  - "Azure.Messaging.WebPubSub"
+- Activity source names:
+  - "Azure.Messaging.WebPubSub.*"
```

Comment:
This doesn't look right to me. When I look at the traces that are coming out of the WebPubSub service client, I see them all prefixed with just `WebPubSubServiceClient`, no `Azure.Messaging`.

![image](https://github.com/dotnet/aspire/assets/8291187/ae50dca5-e305-42da-8d5a-bbafb610561b)

cc @lmolkova

Reply:
these are the span names, they shouldn't contain `Azure.*` prefix. Am I missing something?

Reply:
you're right. This was my mistake. I see it working correctly now.